jobs/sentiment-analysis/sentiment_analysis.py

```diff
@@ -43,20 +65,21 @@ def update_sentiment():
     """Update sentiment by querying the records."""
     conn = None
     try:
+        log_info("Starting sentiment analysis.")
         # connect to the PostgreSQL server
-        conn = psycopg2.connect(**APP_CONFIG.DB_PG_CONFIG)
+        conn = client.connect(Databse[APP_CONFIG.DBMS].value, APP_CONFIG)
 
         table_name = APP_CONFIG.DATABASE_TABLE_NAME
         input_col = APP_CONFIG.DATABASE_INPUT_COLUMN
         output_col = APP_CONFIG.DATABASE_OUTPUT_COLUMN
 
         # Find primary key for the table.
         primary_keys = _find_primary_keys(conn, table_name)
-
+        log_info(f"found primary keys : {primary_keys}")
         # Query the rows from table.
         cols_to_query = f'{primary_keys},{input_col}'
-        rows_query = f"select {cols_to_query} from {table_name} where coalesce({output_col}, '') = ''"
-
+        rows_query = f"select {cols_to_query} from {table_name} where coalesce({output_col}, '') = '' limit 100"
```

Comment:
Setting the limit seems causing issues in SQL Server @sumesh-aot looking into it